### ISSUE TITLE
assertion failed: ticks < minar:: platform::Time_Mask

### ISSUE THREAD

**Issue by cyliangtw** [ARMmbed/sal-iface-6lowpan#15]
0xffffffff seems not one valid input value of minar::milliseconds(), so TCPAsynch::TCPAsynch will encounter the assertion, please check below snippet of ns_sal.c: 
```
uint32_t ns_sal_socket_periodic_interval(...)
{
    if (SOCKET_STREAM == socket->family) {
        return 0xffffffff;
    }
    return 0;
}

````

**Comment by ciarmcom** [comment]
ARM Internal Ref: ONME-2556

**Comment by artokin** [comment]
Commit eeabdd6aefa93a1167f97e06782f5e287dc93ab9 will fix this. Published in new patch #2.1.1.
